src/index.flow.js

```diff
@@ -2,12 +2,12 @@
 
 import { type EdgeFetchFunction, type EdgeLog } from 'edge-core-js'
 
-type CreateRepoOptions = {
+type OptionParams = {
```

Comment:
All our other repos use `XyzOptions` types for the final options argument, so we should use something like that here. `EnvironmentOptions`? `CommonOptions`?